src/mesh/stencil/StencilDataBase.cpp

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 NeoFOAM authors
+
+#include "NeoFOAM/mesh/stencil/StencilDataBase.hpp"
+
+std::any& NeoFOAM::StencilDataBase::operator[](const std::string& key)
+{
+    return stencilDB_.at(key);
+}
+
+const std::any& NeoFOAM::StencilDataBase::operator[](const std::string& key) const
+{
+    return stencilDB_.at(key);
+}
+
+bool NeoFOAM::StencilDataBase::contains(const std::string& key) const
+{
+    return stencilDB_.find(key) != stencilDB_.end();
```

Comment:
C++20 has a contains member. Again I think that could be moved to Dictionary and we can rid of StencilDataBase entirely.

```suggestion
    return stencilDB_.contains(key);
```